core/storage/file/gae_models.py

```diff
@@ -47,6 +47,11 @@ class FileMetadataModel(base_models.VersionedModel):
 
     @classmethod
     def get_new_id(cls, entity_name):
+        """Base class to be implemented in sub classes.
```

Comment:
This is not a class. It is a classmethod.

Reply:
Done!